.github/workflows/ci-orchestrator.yml

```
@@ -93,6 +93,25 @@ jobs:
+          # 0. Check for no-auto-merge label
+          echo "Checking for no-auto-merge label..."
+          HAS_NO_AUTO_MERGE=$(gh pr view "$PR_NUMBER" --json labels --jq '.labels[] | select(.name == "no-auto-merge") | .name' | head -n 1)
+          if [[ -n "$HAS_NO_AUTO_MERGE" ]]; then
+             echo "::notice::PR has 'no-auto-merge' label. Skipping auto-merge."
+             # Check if comment already exists to avoid duplicates
+             COMMENT_EXISTS=$(gh api "repos/$REPO/issues/$PR_NUMBER/comments" --jq '.[] | select(.body | contains("Auto-Merge Skipped") and contains("no-auto-merge")) | .id' | head -n 1)
+             if [[ -z "$COMMENT_EXISTS" ]]; then
+               # Post a comment explaining why auto-merge was skipped (only if not already posted)
+               gh pr comment "$PR_NUMBER" --body "🚫 **Auto-Merge Skipped**
+
+This PR has the \`no-auto-merge\` label. Auto-merge was not executed.
```

actionlint (1.7.10)

could not parse as YAML: could not find expected ':'

(syntax-check)